### PR TITLE
Add multiple choices field type

### PR DESCRIPTION
**Pull request by leplatrem**

### CODE REVIEW

**Comment by AntoineCezar** [comment]
Works for me

File: daybed/schemas/base.py [hunk 1]
```diff
@@ -180,6 +184,39 @@ def validation(cls, **kwargs):
         return super(EnumField, cls).validation(**kwargs)
 
 
+class JSONList(List):
+    """A set of items in JSON-like format"""
```
Comment (almet) [inline daybed/schemas/base.py]: Why json _like_?

File: daybed/tests/test_types.py [hunk 2]
```diff
@@ -74,6 +74,39 @@ def test_optional_field(self):
...
+
+    def test_choices(self):
+        schema = schemas.ChoicesField.definition()
+        definition = schema.deserialize(
+            {'name': 'tags',
+             'type': 'choices',
+             'choices': ['a', 'b', 'c']})
+
+        validator = schemas.ChoicesField.validation(**definition)
+        self.assertEquals(['a'], validator.deserialize('a'))
```
Comment (almet) [inline daybed/tests/test_types.py:97]: I have share thoughts about the data we should be able to recognize. What's the rationale behind having non-json data?

**Comment by almet** [comment]
This works for me. I would like to have some reasoning about the _like_ part. When would we want to have stuff looking like JSON but which isn't JSON?

**Comment by leplatrem** [comment]
The only use-case I have in mind is with posting forms values.
For example, in most geometry editing applications, we fill a hidden textarea field with serialized JSON inside. When form is posted, the server receives the JSON data as string.

I also thought it would be an easy way to give multiple choices, as comma-separated : `values = "1,2,3"`

Hence, that "JSON-like" comment should be changed into something like "Pure JSON or string, as serialized JSON or comma-separated values"

**Comment by almet** [comment]
+1

**Comment by almet** [comment]
updated + merged.

**Pull request merged**